integrated_channels/canvas/dev_docs/postman_collections/Canvas.postman_collection.json

```diff
@@ -58,17 +58,17 @@
 						},
 						{
 							"key": "client_id",
-							"value": "10000000000002",
+							"value": "{{client_id}}",
 							"type": "text"
 						},
 						{
 							"key": "client_secret",
-							"value": "bSxFhGIBmqluIMe7fIc0S6fNU75WtD7sB7r6T8euVwoYJ6eeNZIU0cfBS4VwjHwJ",
+							"value": "{{client_secret}}",
 							"type": "text"
 						},
 						{
 							"key": "code",
-							"value": "b77ce731852e1691c69eac9ac5bf8d62bdf8ae5f59c521726e37e0aa87ab49a6561b615146ff918e0bd79556c055a81f9fce589e35c5bf98eaafa88f703939bf",
+							"value": "{{code}}",
```

Comment:
Just a heads up that this var isn't mentioned in the readme.

Reply:
nice catch got it!